cmd/frontend/auth/redirect.go

```diff
@@ -15,6 +15,9 @@ func SafeRedirectURL(urlStr string) string {
 		return "/"
 	}
 
+	// Make sure u.Path always starts with a single slash.
+	u.Path = "/" + strings.TrimLeft(u.Path, "/")
```

Comment:
Alternatively this does a bit more:
```suggestion
	u.Path = path.Clean(u.Path)
```

Reply:
Nice, thanks!